src/components/input/index.tsx

```diff
@@ -99,13 +104,38 @@ const Input = forwardRef<HTMLInputElement, Props>(
       visible: !!value,
       disabled,
       onClear: () => {
+        ;(inputRef.current as HTMLInputElement).value = ""
         onChange({ target: { name, value: "", cleared: true } })
       },
     }
 
+    const debouncedChangeHandler = useMemo(() => {
+      if (onChange) {
+        return debounce(onChange, delay)
+      }
+      return null
+    }, [])
```

Comment:
would we want to include onChange or delay in the deps? they'd never change in practice I guess...

Reply:
Yea, in practice those wont change, but its still nice to have them 👍